games/openrct2/Portfile

```diff
@@ -43,9 +73,42 @@ depends_lib-append  port:libsdl2 \
                     port:libzip \
                     port:curl \
                     port:freetype \
-                    port:libiconv
+                    port:libiconv \
+                    port:nlohmann-json
+
+compiler.cxx_standard       2017
+configure.cxxflags-append   -std=c++17
+
+post-extract {
+    file mkdir ${worksrcpath}/data/sequence
+    move {*}[glob -nocomplain ${workpath}/*.parkseq] \
+        ${worksrcpath}/data/sequence
+    file mkdir ${worksrcpath}/data/object
+    move ${workpath}/official {*}[glob -nocomplain ${workpath}/rct*] \
+        ${worksrcpath}/data/object
+}
+
+patch {
+    reinplace "s|git describe HEAD|echo v${version}|" CMakeLists.txt
+    reinplace "s|git rev-parse --abbrev-ref HEAD|echo master|" CMakeLists.txt
+    reinplace "s|git rev-parse --short HEAD|echo 0000000|" CMakeLists.txt
+    reinplace "s|<nlohmann/json_fwd.hpp>|<nlohmann/json.hpp>|" \
+        src/openrct2/core/JsonFwd.hpp
```

Comment:
This type of thing should be done as a patchfile, not a reinplace, and again it looks like an upstream issue, so add an upstream bug report URL so that we can track the progress of upstream fixing the problem so that we can eventually remove our patch.